napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -210,7 +214,7 @@ def __init__(self, layer):
             'napari:activate_label_erase_mode',
             self.erase_button,
             extra_tooltip_text=trans._(
-                "Toggle with {shortcut}",
+                "\n(or {shortcut})",
                 shortcut=Shortcut("Alt"),
             ),
```

Comment:
Same as above. This shortcut doesn't work at present and with #5018 it will be via the UI and not hard-coded. Suggest deleting.

Reply:
Deleted